src/ServiceManager.js

```diff
@@ -0,0 +1,45 @@
+const boxen = require('boxen')
+const chalk = require('chalk').default
+const bonjour = require('nbonjour').create()
+const name = process.env.LIVING_ROOM_NAME || require('os').hostname()
```

Comment:
Unless I'm reading this wrong, you're not using this... should that be setting `this.name` when you instantiate `serviceManager` in `service`?

Reply:
good catch, ServiceManager shouldn't even know what the name is tbh